src/OpenTelemetry.Instrumentation.StackExchangeRedis/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## 1.0.0-rc9.6
```

Comment:
Looks like we also typically add the date below the version would you mind also adding that?